src/commands/aksKubectlCommands/aksKubectlCommands.ts

```diff
@@ -130,10 +129,6 @@ async function loadKubectlCommandRun(
   kubectl: k8s.APIAvailable<k8s.KubectlV1>) {
 
   const clustername = cloudTarget.name;
-  // Download and get Kubelogin path for AAD Enabled AKS Cluster.
-  // Sample use
-  const kubeloginPath = await getKubeloginBinaryPath();
```

Comment:
💡Sounds good, Adding for archival reasons: `getAadKubeconfig` is declared under `cluster.ts` based on `aadProfile` property.